README.md

```diff
@@ -44,6 +44,32 @@ example, you can try the `vector addition` example as follows:
 python numba_dppy/examples/sum.py
 ```
 
+# Known Issue
```

Comment:
I think there should be a sections about limitations in documentation.
I.e. there is example for device function where limiataions are - todo - https://github.com/IntelPython/numba-dppy/blame/main/docs/user_guides/kernel_programming_guide/device-functions.rst#L29
I think this limitation is related to Python features support so the better document is https://github.com/IntelPython/numba-dppy/blob/main/docs/user_guides/kernel_programming_guide/supported-python-features.rst

Reply:
@reazulhoque 
You can keep the section `Known Issues` and place the link to corresponding docs inside it. 
It is more scalable approach as we can not place all limitations in README.